io.github.everestapi.Olympus.yml

```diff
@@ -0,0 +1,61 @@
+app-id: io.github.everestapi.Olympus
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: olympus
+finish-args:
+  # Allow the GUI to render
+  - --socket=x11
+  - --share=ipc
+  # Needed to access mod list and download mods
+  - --share=network
+  # Allow OpenGL acceleration
+  - --device=dri
+  # Needed to get the sound working
+  - --socket=pulseaudio
+  # Needed to launch celeste from Olympus
+  - --talk-name=org.freedesktop.Flatpak
+  # Keep the usual functions of Olympus working
+  - --filesystem=home
```

Comment:
Why do you need full home?

Reply:
Is there a specific location these folders can be in like `~/.steam` or something?

Reply:
It can have specific path like the steam one however in the modding community many people also have itch.io drm free builds that are not in any specific paths. Thus in order to keep it functioning as usual home access is required

Reply:
Itch also has a specific location, doesn't it?

(It doesn't matter that much because you're using org.freedesktop.Flatpak which nullifies any filesystem restrictions)

Reply:
Itch has a specific location if using the default installer however it's also possible to clone the game folder and/or downloading only the game directly from the website